src/flowTypes.js

```diff
@@ -410,7 +410,7 @@ export type FileAccessStats = {
 };
 
 export type Task = {
-    type: string,
+    type: 'task',
```

Comment:
is it normal to put a value in there instead of the type of that value?

Reply:
We tend to do it for types that are constant. Look at User, SkillCard, etc